### PR TITLE
(DOCSP-10138) - adds note about twilio whatsapp api

### PR DESCRIPTION
**Pull request by terakilobyte**
[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-10138/services/twilio-actions/send#usage)

### CODE REVIEW

File: source/services/twilio-actions/send.txt [hunk 1]
```diff
@@ -20,6 +20,16 @@ Sends an SMS text message with Twilio.
 Usage
 -----
 
+.. note::
+
+   To send and receive messages via the `Twilio API for WhatsApp <https://www.twilio.com/whatsapp>`__,
```
Comment (cbush) [inline source/services/twilio-actions/send.txt:25]:
```suggestion
   To send or receive messages via the `Twilio API for WhatsApp <https://www.twilio.com/whatsapp>`__,
```

File: source/services/twilio-actions/send.txt [hunk 2]
```diff
@@ -20,6 +20,16 @@ Sends an SMS text message with Twilio.
...
+   prepend the ``to`` and/or ``from`` numbers with ``whatsapp:``.
```
Comment (cbush) [inline source/services/twilio-actions/send.txt:26]:
```suggestion
   prepend the ``to`` or ``from`` numbers with ``whatsapp:``.
```

File: source/services/twilio-actions/send.txt [hunk 3]
```diff
@@ -20,6 +20,16 @@ Sends an SMS text message with Twilio.
...
+
+   .. code-block:: javascript
+
+      to: "whatsapp:+15558675309",
```
Comment (cbush) [inline source/services/twilio-actions/send.txt:30]: Jenny?!

Comment (terakilobyte) [reply source/services/twilio-actions/send.txt:30]: Turns out she loves talking on the phone!

**Pull request merged**